src/battlesnake.theserverproject.com/insert_clever_name/snake/engine/moveutils.go

```diff
@@ -0,0 +1,122 @@
+package engine
+
+import (
+	"battlesnake.theserverproject.com/insert_clever_name/snake/game"
+	"errors"
+)
+
+const (
+	UP    string    = "up"
+	DOWN  string    = "down"
+	LEFT  string    = "left"
+	RIGHT string    = "right"
+	OPPONENT string = "o"
+	NONE  string    = "n"
+)
+
+var MOVES = [...]string{UP, DOWN, LEFT, RIGHT}
+var OPPOSITE_MOVE = map[string]string{
+	UP:    DOWN,
+	DOWN:  UP,
+	LEFT:  RIGHT,
+	RIGHT: LEFT,
+}
+
+func checkMyMove(g game.Game, coord game.Coordinate) (bool, error) {
+	valid := true
```

Comment:
The `valid` variable seems very unneeded here. Each case could simply return `true, nil`, or `false, nil`. Simplifies the code I believe removing this.

Reply:
Good point and I agree, will change.